packages/aws-cdk/test/api/test.bootstrap.ts

```diff
@@ -144,7 +144,57 @@ export = {
       region: 'us-east-1',
       name: 'mock',
     }, sdk, 'mockStack', undefined, {
-      kmsKeyId: 'myKmsKey',
+      kmsKeyId: 'myKmsKey', tags: undefined
+    });
+
+    // THEN
+    test.equals(ret.noOp, false);
+    test.equals(executed, true);
+
+    test.done();
+  },
+  async 'do bootstrap using tags'(test: Test) {
```

Comment:
how do bootstrap with tags for CDK Toolkit stack?

...or anything that helps clarify that the tags are on the stack.